### PR TITLE
[bug] : fix footer alignment issue #266

### PR DESCRIPTION
**Pull request by nxjiten**
@natabal @killanch 

Fix issue #266. Please have a look.

### CODE REVIEW

**Comment by killanch** [comment]
@nxjiten I have updated https://github.com/nuagenetworks/vis-graphs/issues/266 with 2 other issues I noticed with these latest changes.
Could you pls address them as well?

**Comment by nxjiten** [comment]
@killanch @natabal 
Fix all issues #266. Please have a look.

File: Graphs/Table/index.js [hunk 1]
```diff
@@ -806,13 +815,17 @@ class Table extends AbstractGraph {
     // reset scroll data.
     resetScrollData() {
         const { disableRefresh } = this.getConfiguredProperties();
+        const {
+            isVSDService,
```
Comment (natabal) [inline Graphs/Table/index.js]: This is a lib that should not be aware of what service this is! Please provide a way of injecting styles, similarly to how material-ui is using withStyles

**Comment by nxjiten** [comment]
@natabal @killanch 
Fix all the latest issues in #266 . Please have a look

File: Graphs/Table/index.js [hunk 2]
```diff
@@ -960,7 +965,7 @@ class Table extends AbstractGraph {
 
         const rowsPerPageSizes = uniq([10, 15, 20, 100, pageSize]);
         const rowsPerPageOptions = rowsPerPageSizes.filter(rowsPerPageSize => rowsPerPageSize < totalRecords);
-        const showFooter = (totalRecords <= pageSize &&  totalRecords !== 0 && hidePagination !== false) ? false : true;
+        const showFooter = (totalRecords <= pageSize && hidePagination !== false && scroll != true) ? false : true;
```
Comment (natabal) [inline Graphs/Table/index.js]: please use here !==

**Comment by nxjiten** [comment]
@natabal @killanch 
Fixed all the issues. Please have a look.

**Pull request merged**